postSkybill.php

```diff
@@ -11,7 +11,7 @@
 */
 
 if (!defined('_MYDIR_')) define('_MYDIR_', dirname(__FILE__));
-require('../../config/config.inc.php');
+require($_SERVER['DOCUMENT_ROOT'].'/config/config.inc.php');
```

Comment:
We can go back to original code for this: I used it because the 
development project is symlinked to.

On 06/01/2016 12:56 PM, francoisfeugeas wrote:
>
> In postSkybill.php 
> <https://github.com/PrestaShop/chronopost/pull/36#discussion_r65340091>:
>
> > @@ -11,7 +11,7 @@
> >  */
> >
> >  if (!defined('_MYDIR_')) define('_MYDIR_', dirname(__FILE__));
> > -require('../../config/config.inc.php');
> > +require($_SERVER['DOCUMENT_ROOT'].'/config/config.inc.php');
>
> This won't work in most deployments cases !
> Can you resubmit the PR using |_PS_ROOT_DIR_| instead ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/PrestaShop/chronopost/pull/36/files/535e5bbe0bc6303d962f45148a0662bc1375a761#r65340091>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AC4s479Lvd9Rat2GJ3VfhUDx2LO8M29Pks5qHWVdgaJpZM4Iraav>.
>